### PR TITLE
Fix: move "bridge.yaml" config file logic to "config.yaml"

### LINKED ISSUE #978: Fix: 0chain smart contract logs are not shown

### PR DESCRIPTION
**Pull request by YarikRevich**
## Associated PRs:
- https://github.com/0chain/zwalletcli/pull/253
- https://github.com/0chain/system_test/pull/688
- https://github.com/0chain/actions/pull/188
- https://github.com/0chain/0helm/pull/265
- https://github.com/0chain/gitbookdocs/pull/9

### CODE REVIEW

File: wasmsdk/bridge.go [hunk 1]
```diff
@@ -12,60 +12,34 @@ import (
...
-//}
-
 var bridge *zcnbridge.BridgeClient
 
 func initBridge(
 	ethereumAddress string,
 	bridgeAddress string,
 	authorizersAddress string,
-	wzcnAddress string,
+	TokenAddress string,
```
Comment (peterlimg) [inline wasmsdk/bridge.go]: let's use tokenAddress to keep all params name in consistent format

File: zcnbridge/config.go [hunk 2]
```diff
@@ -1,260 +1,97 @@
...
-		Instance: &Instance{
-			startTime: common.Now(),
-			zcnWallet: wallet,
-		},
-	}
+	ConsensusThreshold float64
+	GasLimit           uint64
 }
 
 func CreateBridgeClient(cfg *viper.Viper, walletFile ...string) *BridgeClient {
```
Comment (peterlimg) [inline zcnbridge/config.go]: The walletFile option param is not being used

File: zcnbridge/config.go [hunk 3]
```diff
@@ -1,260 +1,97 @@
...
-	err = json.Unmarshal(clientBytes, &wallet)
+func readConfig(sdkConfig *BridgeSDKConfig, getConfigName func() string) *viper.Viper {
+	cfg := viper.New()
+	cfg.AddConfigPath(*sdkConfig.ConfigDir)
+	cfg.SetConfigName(getConfigName())
+	cfg.SetConfigType("yaml")
+	err := cfg.ReadInConfig()
 	if err != nil {
-		return nil, err
+		_, file, line, ok := runtime.Caller(2)
```
Comment (peterlimg) [inline zcnbridge/config.go]: do we really need to return the file name and line number when the error happens. Knowing the err should be enough, if we really need to know where the  error happens, give more info in the error message to differ it from common error should be enough as well.

File: zcnbridge/keystore.go [hunk 4]
```diff
@@ -80,30 +78,30 @@ func CreateKeyStorage(homedir, password string) error {
...
-	text, err := yaml.Marshal(cfg)
-	if err != nil {
-		return err
-	}
-
-	err = os.WriteFile(configFile, text, 0644)
-
-	return err
-}
+// // UpdateClientEthereumAddress updates Ethereum address
```
Comment (peterlimg) [inline zcnbridge/keystore.go]: Remove commented code if not being used

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: wasmsdk/bridge.go (hunk 1 above)
Comment (YarikRevich) [reply wasmsdk/bridge.go]: Done

File: zcnbridge/config.go (hunk 2 above)
Comment (YarikRevich) [reply zcnbridge/config.go]: Done

File: zcnbridge/config.go (hunk 3 above)
Comment (YarikRevich) [reply zcnbridge/config.go]: Done

File: zcnbridge/keystore.go (hunk 4 above)
Comment (YarikRevich) [reply zcnbridge/keystore.go]: Done

**Review by peterlimg — APPROVED** [review]
LGTM

**Comment by Kishan-Dhakan** [comment]
Passing run: https://github.com/0chain/gosdk/actions/runs/5143661579/jobs/9258994464?pr=979

**Pull request merged**